src/pages/[language]/[slug]/index.query.graphql

```diff
@@ -92,6 +92,29 @@ query LandingPage($locale: SiteLocale, $slug: String) {
           }
         }
       }
+      ... on SectionImageCardGridRecord {
+        title
+        items {
+          title
+          body
+          image {
+            ...image
+          }
+        }
+      }
+      ... on SectionJobsListRecord {
+        id
+        jobs {
+          title
+          slug
+          jobImage {
+            url
+            alt
+            width
+            height
```

Comment:
Could we use the image fragment here?

Reply:
I didn't find an existing fragment, are you referring to the changes in the `graphql-fragments` branch?

Reply:
There's one already all the way at the bottom of this query file